packages/story-editor/src/app/canvas/canvasProvider.js

```diff
@@ -62,18 +62,20 @@ function CanvasProvider({ children }) {
   // to multiple nodes.
   const clientRectObserver = useMemo(
     () =>
-      new window.IntersectionObserver((entries) => {
-        for (const entry of entries) {
-          if (!entry.target.dataset[RECT_OBSERVATION_KEY]) {
-            return;
-          }
-          setBoundingBoxes((boxes) => ({
-            ...boxes,
-            [entry.target.dataset[RECT_OBSERVATION_KEY]]:
-              entry.boundingClientRect,
-          }));
-        }
-      }),
+      globalThis.window
+        ? new window.IntersectionObserver((entries) => {
```

Comment:
Shouldn't this be like this?

```suggestion
      globalThis.IntersectionObserver
        ? new globalThis.IntersectionObserver((entries) => {
```